plotly/presentation_objs/presentation_objs.py

```diff
@@ -0,0 +1,1199 @@
+                                   'fontWeight': 400,
+                                   'lineHeight': 'normal',
+                                   'minWidth': 20,
+                                   'opacity': 1,
+                                   'textAlign': 'center',
+                                   'textDecoration': 'none'},
+                     'Heading 2': {'color': '#3d3d3d',
+                                   'fontFamily': 'Open Sans',
+                                   'fontSize': 20,
+                                   'fontStyle': 'normal',
+                                   'fontWeight': 400,
+                                   'lineHeight': 'normal',
+                                   'minWidth': 20,
+                                   'opacity': 1,
+                                   'textAlign': 'center',
+                                   'textDecoration': 'none'},
+                     'Heading 3': {'color': '#3d3d3d',
+                                   'fontFamily': 'Open Sans',
+                                   'fontSize': 11,
+                                   'fontStyle': 'normal',
+                                   'fontWeight': 700,
+                                   'lineHeight': 'normal',
+                                   'minWidth': 20,
+                                   'opacity': 1,
+                                   'textAlign': 'center',
+                                   'textDecoration': 'none'}}
+
+
+def _empty_slide(transition, id):
+    empty_slide = {'children': [],
+                   'id': id,
+                   'props': {'style': {}, 'transition': transition}}
+    return empty_slide
+
+
+def _box(boxtype, text_or_url, left, top, height, width, id, props_attr,
+         style_attr, paragraphStyle):
+    children_list = []
+    fontFamily = "Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace"
+    if boxtype == 'Text':
+        children_list = text_or_url.split('\n')
+
+        props = {
+            'isQuote': False,
+            'listType': None,
+            'paragraphStyle': paragraphStyle,
+            'size': 4,
+            'style': copy.deepcopy(_paragraph_styles[paragraphStyle])
+        }
+
+        props['style'].update(
+            {'height': height,
+             'left': left,
+             'top': top,
+             'width': width}
+        )
+
+    elif boxtype == 'Image':
+        props = {
+            'height': 512,
+            'imageName': None,
+            'src': text_or_url,
+            'style': {'height': height,
+                      'left': left,
+                      'opacity': 1,
+                      'position': 'absolute',
+                      'top': top,
+                      'width': width},
+            'width': 512
+        }
+    elif boxtype == 'Plotly':
+        props = {
+            'frameBorder': 0,
+            'scrolling': 'no',
+            'src': text_or_url + '.embed?link=false',
```

Comment:
is this missing `sharekey`?

Reply:
it's not, but the embed line should not be there if there is a sharekey - I just tested that

now it checks if `?sharekey` is in the url and will not put the `.embed?link=false` if that is true

Reply:
Even with a sharekey it should still have `.embed` and `link=false`, it will just look something like:
```
.embed?link=false&sharekey=....
```

vs just
```
.embed?link=false
```